explorer/templates.go

```diff
@@ -600,5 +600,13 @@ func makeTemplateFuncMap(params *chaincfg.Params) template.FuncMap {
 			return intStr
 		},
 		"floor": math.Floor,
+		"getTotalFromRewardWindowRewardWindowSize": func(idx int, max, t int64) int {
```

Comment:
The name "getTotalFromRewardWindowRewardWindowSize" is extremely confusing.  It computes a time/duration, but that's not even suggested by the name.  Plus `RewardWindowRewardWindow`...

In plain language, what does this function do?  It computes the time remaining in the current reward window?